### PR TITLE
Create Slack-Notification.yaml

### PR DESCRIPTION
**Pull request by dereban25**
If failed CI/CD send message to personal slack channel

### CODE REVIEW

File: .github/workflows/Slack-Notification.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Slack-Notification
```
Comment (alonashkenazi) [inline .github/workflows/Slack-Notification.yaml:1]: rename file to slack-notification.yaml (all lower case)

**Comment by alonashkenazi** [comment]
We need to also notify to cloudrail-dev channel when we release a new version (only from main)

**Review by alonashkenazi — CHANGES_REQUESTED** [review]

File: .github/workflows/Slack-Notification.yaml (hunk 1 above)
Comment (dereban25) [reply .github/workflows/Slack-Notification.yaml]: Done.I changed a name of file and added trigger on release.

**Comment by dereban25** [comment]
Added another pull request

**Pull request closed without merging**